### PR TITLE
Bug/820 missing translator comments

### PR DESCRIPTION
**Pull request by ShahAaron**
## Summary

Addresses issue #820 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/element-shim.js [hunk 1]
```diff
@@ -4,8 +4,6 @@
  * A temporary workaround to ensure the same version of React
  * is always used across multiple entrypoints.
  *
- * @private
```
Comment (felixarntz) [inline assets/js/element-shim.js:7]: Why was this removed here?

File: assets/js/hooks-shim.js [hunk 2]
```diff
@@ -4,8 +4,6 @@
  * A temporary workaround to share the same instance of hooks
  * across multiple entrypoints.
  *
- * @private
```
Comment (felixarntz) [inline assets/js/hooks-shim.js:7]: See above.

File: assets/js/element-shim.js (hunk 1 above)
Comment (ShahAaron) [reply assets/js/element-shim.js:7]: @felixarntz it was removed as there was a linter error generated from it. I couldn't find a work around, would you like me to look into getting the linter to ignore that line?

File: assets/js/hooks-shim.js (hunk 2 above)
Comment (ShahAaron) [reply assets/js/hooks-shim.js:7]: As above
